spring-integration-core/src/main/java/org/springframework/integration/config/ConverterRegistrar.java

```diff
@@ -72,7 +74,10 @@ private void registerConverters(GenericConversionService conversionService) {
 						.values()
 						.stream().map(IntegrationConverterRegistration::converter)
 						.collect(Collectors.toSet());
-		if (JacksonPresent.isJackson2Present()) {
+		if (JacksonPresent.isJackson3Present()) {
+			converters.add(new JsonNodeWrapperConverter());
+		}
+		else if (JacksonPresent.isJackson2Present()) {
```

Comment:
I think this is similar to the previous one.
We do not give a choice to end-user here.
Better to have both converters registered for time being until we remove those deprecated classes.
But yeah, Jackson 3 has to be first as priority.